lib/conjur/fpm/package.sh

```diff
@@ -22,6 +22,10 @@ for i in "$@"; do
     file_type="${i#*=}"
     shift
     ;;
+  -fa=* | --architecture=*)
```

Comment:
Is the "f" in "-fa" significant here? If not, I might suggest only having the long form and not a short form. I'm not sure seeing "-fa" provides a reasonable affordance that this abbreviates "architecture".

Reply:
Good point, i refactored it to the full flag only